src/pages/ReportDetailsPage.tsx

```diff
@@ -332,7 +354,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                             />
                         </OfflineWithFeedback>
                     )}
-                    {isGroupChat && <ChatDetailsQuickActionsBar report={report} />}
+                    <ChatDetailsQuickActionsBar report={report} />
```

Comment:
@ZhenjaHorbach From my viewpoint, we should update like this
```suggestion
                    {!isGroupDMChat && <ChatDetailsQuickActionsBar report={report} />}
```

To make sure the logic to display the share button is same like before